Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -173,6 +181,26 @@ public class PBXProjGenerator {
 
         let configs: [XCBuildConfiguration] = spec.configs.map { config in
             var buildSettings = spec.getTargetBuildSettings(target: target, config: config)
+            // Do not overwrite target's xcconfig values.
+            if let configPath = spec.targets.flatMap({ $0 == target ? $0.configFiles[config.name] : nil }).first {
+                if let configFile = try? XCConfig(path: basePath + configPath) {
+                    for (k, _) in configFile.flattenedBuildSettings() {
+                        // FIXME: Catch platform specifier. e.g. LD_RUNPATH_SEARCH_PATHS[sdk=iphone*]
+                        buildSettings.removeValue(forKey: k)
+                        buildSettings.removeValue(forKey: "\"\(k)\"")
+                    }
+                }
+            }
+            // Do not overwrite base xcconfig values.
+            if let configPath = spec.configFiles[config.name] {
```

Comment:
I don't think we have to remove target build settings because of a project config file